src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Truck truck = new Truck();
+        Excavator excavator = new Excavator();
+        Bulldozer bulldozer = new Bulldozer();
```

Comment:
Instead of declaring the objects with their concrete classes, use the abstract class reference. This will allow for more flexible code that can work with any subclass of Machine.